.eslintrc

```diff
@@ -26,8 +26,7 @@
       "error",
       {
         "code": 120,
-        "ignoreUrls": true,
-        "ignorePattern": "*className=([\\s\\S]*?)*" // Ignore classnames
```

Comment:
IMO we should disable this max-len rule. I disabled it in my [PR](https://github.com/asyncapi/website/pull/2734/files#r1511461780) since some `classNames` were greater than 120 characters but there was still an error.

Though, is there any example case where max-len rule would actually help us?

Reply:
Yeah, I was facing a lot of problems too. Hence I had added this in my previous PR. However as eslint was still throwing errors, I removed it.

Sth can be done about max-len though as tailwind classNames get really big.